readme.md

```diff
@@ -124,17 +124,23 @@ If you don't have root access, just install everything on your user's folder and
 
 This is an example on how to record the system update package count daily.  
 This will update the file `/var/tmp/updatecount.txt` for later access.  
-Root privilege is required for this operation.
+Root privilege is required for this operation. 
+Only /etc/crontab and the files in /etc/cron.d/ have a username field.
+ 
+##### So there you can do this:
 
-##### To edit root's crontab
 ```bash
-sudo crontab -e
+sudo vim /etc/crontab
```

Comment:
Honestly I'd not recommend editing the crontab directly. On my system these lines are inside `/etc/cron.d/firemotd`